src/Microsoft.Extensions.Testing.Abstractions/Messages/Message.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Extensions.Testing.Abstractions
+{
+    public class Message
+    {
+        public string MessageType { get; set; }
+
+        public JToken Payload { get; set; }
```

Comment:
I'm avoiding changes to the interface between VS and dotnet-test [as it's the same as was in DNX test].